{"uuid":"b977f81d-b40a-4a5f-b8ca-7757d28e7e55","historyId":"8d8f2dcbd981ee6e22490f5553b8cc8e","fullName":"example.UserProfileManagement.UPM_02_05_VerifyUpdateProfileSuccess.VerifyUpdateProfileSuccess","labels":[{"name":"package","value":"example.UserProfileManagement.UPM_02_05_VerifyUpdateProfileSuccess"},{"name":"testClass","value":"example.UserProfileManagement.UPM_02_05_VerifyUpdateProfileSuccess"},{"name":"testMethod","value":"VerifyUpdateProfileSuccess"},{"name":"parentSuite","value":"CRM Automation"},{"name":"suite","value":"User Profile Management"},{"name":"subSuite","value":"example.UserProfileManagement.UPM_02_05_VerifyUpdateProfileSuccess"},{"name":"host","value":"LAPTOP-4T54UG83"},{"name":"thread","value":"10792@LAPTOP-4T54UG83.TestNG-tests-2(18)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"VerifyUpdateProfileSuccess","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tUpdate error expected [Vy] but found [Thy]","trace":"java.lang.AssertionError: The following asserts failed:\n\tUpdate error expected [Vy] but found [Thy]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat example.UserProfileManagement.UPM_02_05_VerifyUpdateProfileSuccess.VerifyUpdateProfileSuccess(UPM_02_05_VerifyUpdateProfileSuccess.java:46)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.lang.Thread.run(Thread.java:829)\r\n"},"stage":"finished","steps":[{"name":"Open CRM Website","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724340263889,"stop":1724340263889},{"name":"Login function","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724340264022,"stop":1724340264022},{"name":"Update profile with proper values","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724340266966,"stop":1724340266966},{"name":"Check update success","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724340268363,"stop":1724340268363}],"attachments":[],"parameters":[],"start":1724340263889,"stop":1724340268416}